examples/sglang_bench/bench_sharegpt.py

```
@@ -110,6 +119,108 @@ def detect_tp_size() -> Optional[int]:
+
+    python_cmd = ["python", "-m", "sglang.launch_server"] + worker_args
+    cmd_str = " ".join(shlex.quote(c) for c in python_cmd)
+
+    if conda_env:
+        remote_cmd = f"conda run --no-capture-output -n {shlex.quote(conda_env)} {cmd_str}"
+    else:
+        remote_cmd = cmd_str
+
+    ssh_cmd = ["ssh", "-o", "StrictHostKeyChecking=no", "-o", "BatchMode=yes", peer_host, remote_cmd]
+    log(f"Launching peer worker (rank {node_rank}) on {peer_host}")
+    return subprocess.Popen(ssh_cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)